src/main/java/core/basesyntax/model/Gambler.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public class Gambler {
```

Comment:
I would suggest to implement specific `GamblerConsoleHandler` class in scope of this application. As we can save `bets` we might also want to save `gamblers`.

Reply:
done